### PR TITLE
Introduce internal kep to the project

### LINKED ISSUE #173: internal KEP process for this repo
Like scheduler-plugin, I'd like to introduce KEP process for this repo.
https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/kep

- [ ] create `/kep` dir
- [ ] add doc (What is KEP? What kind of features needs KEP? How to write that? Where is the template?)

/assign
/kind documentation

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Introduce internal kep to the project

#### Which issue(s) this PR fixes:

Fixes #173 

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ Welcome to Kubernetes. We are excited about the prospect of you joining our [com
 
 _As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities._
 
+- [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
```
Comment (196Ikuchil) [inline CONTRIBUTING.md:7]:
```suggestion
- [Contributor License Agreement](https://git.k8s.io/community/CLA.md) - Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,10 +31,7 @@ make format
 
 see [Makefile](Makefile) for more details.
```
Comment (196Ikuchil) [inline CONTRIBUTING.md:32]: nit.
```suggestion
See [Makefile](Makefile) for more details.
```

**Review by 196Ikuchil — COMMENTED** [review]
@sanposhiho 
Thanks, I left some comments!

**Comment by sanposhiho** [comment]
@196Ikuchil Thanks, fixed.

**Comment by 196Ikuchil** [comment]
Thanks!
/lgtm

**Pull request merged**